src/main/resources/org/sakaiproject/feedback.properties

```diff
@@ -30,7 +30,7 @@ content_instruction=Use this page to report a problem or issue with this site's
 <li>permission denied errors or access problems</li>\
 <li>issues with dates in calendar</li>\
 </ul>
-technical_instruction=Assuming you cannot find the answer in the <a href=\\"/portal/help/\\" target=\\"_blank\\">{0} help pages</a>, \
```

Comment:
I don't this this fixes it as it's still loading the local instructions from the en_GB file.